src/lib/denoify.ts

```diff
@@ -58,23 +59,6 @@ export async function denoify(
     })();
 
 
-    if (!("main" in packageJsonParsed)) {
-        //TODO: We shouldn't force users to specify a default export.
-        throw new Error([
-            "A main field in package.json need to be specified",
-            "otherwise we don't know what file the mod.ts should export."
-        ].join(" "));
-    }
-
-    if (
-        !isInsideOrIsDir({
-            "dirPath": tsconfigOutDir,
-            "fileOrDirPath": path.normalize(packageJsonParsed["main"])
-        })
-    ) {
-        throw new Error(`The package.json main should point to a file inside ${tsconfigOutDir}`)
-    }
-
```

Comment:
I no longer have all the context in mind to remember why I deemed important to have this fail early strategy here.  
I am willing to trust you on the fact that it can be safely removed.